lib/routes.dart

```diff
@@ -403,6 +404,14 @@ final List<RouteBase> routes = [
       child: const LoginScreen(),
     ),
   ),
+  GoRoute(
+    path: '/liked',
+    name: 'liked',
```

Comment:
Since this view is reachable only from `/albums`, I think it would make sense to make it a child of the albums route? It's not ideal because of a naming conflict with the slug param on `/albums/<slug>`, but on the website, `/members/photos/liked` also shadows a hypothetical album with "liked" as slug, so it doesn't really matter. 

Additionally, it would be nice to support the deeplink (add a regex to _deepLinkRegExps above and ensure the URL is redirected).